commands/utils/docker-endpoint.ts

```diff
@@ -5,14 +5,19 @@
 
 import * as Docker from 'dockerode';
 import * as vscode from "vscode";
+import { nonNullValue } from '../../utils/nonNull';
 
 export enum DockerEngineType {
     Linux,
     Windows
 }
 
 class DockerClient {
-    private endPoint: Docker;
+    private _endPoint: Docker | undefined;
+
+    private get endPoint(): Docker {
+        return nonNullValue(this._endPoint, 'endPoint');
```

Comment:
This could throw an internal error, correct? Where is that being handled?

Reply:
It's not.  Assuming it's inside an command handler, vscode will display any unhandled exceptions (and we'll send telementry).